generate_licenses.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
why, oh why can't it figure out this is the same file? :(

Reply:
Too much changes I'm afraid. I was too drastic :-)